### PR TITLE
Add a toml linter

### PR DESCRIPTION
**Pull request by sambooo**
Supports stdin/stdout and also writing to files. Errors will be printed to stderr so they don't get piped to files unintentionally.

Based on #69

### CODE REVIEW

File: cmd/tomll/main.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	flag.Parse()
+	// read from stdin and print to stdout
+	if flag.NArg() == 0 {
+		s, err := lintReader(os.Stdin)
+		if err != nil {
+			io.WriteString(os.Stderr, err.Error())
+			os.Exit(-1)
+		}
+		io.WriteString(os.Stdout, s)
+		os.Exit(0)
```
Comment (pelletier) [inline cmd/tomll/main.go]: I'd rather see a `if/else` block here rather than `os.Exit(0)`.

**Comment by pelletier** [comment]
That's great! Do you mind adding a `--help` flag that describe the purpose and usage of the program? I think it's important to mention that files will be modified in place.

**Comment by sambooo** [comment]
Usage() is called here by flag when -h or some invalid flags are passed

**Comment by pelletier** [comment]
Perfect. Thanks for your contribution!

**Pull request merged**